### PR TITLE
HPCC-30240 ES Translations for 9.4.X

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations added for Spanish. (ES)

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src/nls/es/hpcc.ts [hunk 1]
```diff
@@ -63,13 +63,16 @@ export = {
...
     AttributesAreRequired: "Atributos son necesarios",
     Audience: "Audiencia",
+    Audit: 'Auditoría',
     AuditLogs: "Registros de auditoría",
     AutoRefresh: "Actualizar Automaticamente",
     AutoRefreshEvery: "Auto-refrescar cada x minutos",
     AutoRefreshIncrement: "Incrementar auto-refresco",
     Back: "Atras",
     BannerColor: "Color de Banner",
+    BannerColorTooltip: 'Cambiar el color de fondo de la barra de navegación superior',
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:73]: Cambiar el color **del**

File: esp/src/src/nls/es/hpcc.ts [hunk 2]
```diff
@@ -169,9 +174,11 @@ export = {
     Critical: "Crítico",
     CrossTab: "Tabulación cruzada",
     CSV: "CSV",
+    CustomLogColumns: 'Columnas de Registro Personalizado',
     Dali: "Dali",
     DaliAdmin: "Administrador de Dalí",
     DaliIP: "IP de Dali",
+    DaliPromptConfirm: '¿Estás seguro? Las sumisiones de Dali Admin son permanentes y no se pueden deshacer.',
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:181]: This is fine as is, but this form is too intimate: ¿Estás seguro? I'd opt for ¿Está seguro?

File: esp/src/src/nls/es/hpcc.ts [hunk 3]
```diff
@@ -468,7 +475,23 @@ export = {
...
+    LogFilterEventTypeTooltip: 'Incluir solo entradas para el tipo de evento de registro especificado',
+    LogFilterFormatTooltip: 'Formato de archivo de registro: CSV, JSON o XML',
+    LogFilterLineLimitTooltip: 'El número de líneas de registro que se incluirán',
+    LogFilterLineStartFromTooltip: 'Incluir líneas de registro a partir de este número de línea',
+    LogFilterRelativeTimeRangeTooltip: 'Un rango de tiempo que rodea el tiempo de la WU, +/- (en segundos)',
+    LogFilters: 'Filtros de Registro',
+    LogFilterSelectColumnModeTooltip: 'Especificar qué columnas incluir en el archivo de registro',
+    LogFilterSortByTooltip: 'ASC - el más antiguo primero, DESC - el más nuevo primero',
+    LogFilterStartDateTooltip: 'Incluir líneas de registro a partir del inicio del rango de tiempo especificado',
+    LogFilterTimeRequired: 'Elija entre "Desde y Hasta la Fecha" o "Búfer de Tiempo Relativo de Registro"',
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:492]: Confirm the quotes used here are correct

**Review by rpastrana — COMMENTED** [review]
@kunalaswani looks good, only found a couple of suggestions. Also, ensure the correct quotes are used.

File: esp/src/src/nls/es/hpcc.ts (hunk 3 above)
Comment (kunalaswani) [reply esp/src/src/nls/es/hpcc.ts:492]: Corrected all the quotes that needed to be adjusted. Thanks for the catch.

File: esp/src/src/nls/es/hpcc.ts (hunk 2 above)
Comment (kunalaswani) [reply esp/src/src/nls/es/hpcc.ts]: Changes made.

File: esp/src/src/nls/es/hpcc.ts (hunk 3 above)
Comment (rpastrana) [reply esp/src/src/nls/es/hpcc.ts:492]: I still see single quotes encapsulating the string and unescaped double quotes w/in the string. I don't know enough about the syntax, but that seems to be different than the other strings.

**Review by rpastrana — APPROVED** [review]
@kunalaswani everything looks fine but please confirm quotes on L492

**Comment by ghalliday** [comment]
@kunalaswani waiting for your confirmation

**Comment by kunalaswani** [comment]
@ghalliday confirm, the single quotes in line 492 wont cause an issue.

**Pull request merged**